api/controllers/UserController.js

```diff
@@ -37,6 +37,11 @@ var setupReputationQuery = function (req, model) {
     }))
 }
 
+const countTaggedPosts = (userId, tag) =>
+  Search.forPosts({users: [userId], tag})
+  .fetchAll()
```

Comment:
on second view, this is actually really inefficient considering we're going to run this 20 times per page of members, because we're fetching all the supporting data for each post when all we want is the count.

this SQL gets us there:
```
select count(*), user_id from post p, posts_tags pt where p.id = pt.post_id and p.user_id in (21, 42) and pt.tag_id = 3 group by 2;
 count | user_id
-------+---------
     1 |      42
    52 |      21
```
or in knex:
```js
bookshelf.knex('post')
.join('posts_tags', 'post.id', 'posts_tags.post_id')
.where('post.user_id', 'in', [21, 42])
.where('posts_tags.tag_id', 3)
.groupBy('user_id')
.select(bookshelf.knex.raw('count(*), user_id'))
```